interfaces/builtin/raw_usb.go

```diff
@@ -51,6 +51,14 @@ const rawusbConnectedPlugAppArmor = `
 /run/udev/data/b180:*    r, # various USB block devices
 /run/udev/data/c18[089]:* r, # various USB character devices: USB serial converters, etc.
 /run/udev/data/+usb:* r,
+
+# Allow binding/unbind of usb driver from usb devices
+/sys/bus/usb/drivers/usb/bind rw,
+/sys/bus/usb/drivers/usb/unbind rw,
+
+# Allow USB/IP operations
+/sys/bus/usb/drivers/usbip-host/* rw,
```

Comment:
Would this be more appropriate in a `usb-ip` interface instead? The `bluez` interface contains similar entries like:

```
/sys/bus/usb/drivers/btusb/     r,
/sys/bus/usb/drivers/btusb/**   r,
```

Which are not part of `raw-usb` - as such I don't think these really belong here. Also is `w` (write) access definitely required to the `usbip-host` driver parameters? If so can you please detail why?

Reply:
I can test with `r` access, but please see comments above first.